src/containers/GrantsTableWrapper.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+
+import { uniq, map, filter, findIndex, sortBy } from 'lodash';
+
+import { useQuery, gql } from '@apollo/client';
+
+import { stripHtml, extractYear } from '../utils';
+
+import GrantTable from '../components/GrantTable';
+
+const GET_ORGANIZATION_GRANTS = gql`
+  query getOrg($organizationId: String!, $grantsFundedOffset: Int, $grantsReceivedOffset: Int, $limit: Int) {
+    organization(uuid: $organizationId) {
+      grantsFunded (offset: $grantsFundedOffset, limit: $limit, orderBy: uuid) {
+        uuid
+        dateFrom
+        dateTo
+        to {
+          name
+          uuid
+        }
+        amount
+        description
+      }
+      grantsReceived (offset: $grantsReceivedOffset, limit: $limit, orderBy: uuid) {
+        uuid
+        dateFrom
+        dateTo
+        from {
+          name
+          uuid
+        }
+        amount
+        description
+      }
+    }
+  }
+`;
+
+const GrantsTableWrapper = (props) => {
+  const { loading, error, data, fetchMore } = useQuery(GET_ORGANIZATION_GRANTS, {
+    variables: { 
+      organizationId: props.organizationId,
+      grantsFundedOffset: 0,
+      grantsReceivedOffset: 0,
+      limit: 100
+    },
+    notifyOnNetworkStatusChange: true,  // Update 'loading' prop after fetchMore
+  });
+
+  if (loading) return 'Loading...';
+  if (error) return `Error! ${error}`;
+
+  if (!data.organization) return `Failed to load org data!`;
+
+  if (data.organization.grantsFunded.length < props.countGrantsFrom || data.organization.grantsReceived.length < props.countGrantsTo) {
```

Comment:
Could this get a short comment explaining what it's for? Or breaking the conditionals out into variables could serve the same function, something like: 

```js
const moreGrantsFundedToLoad = data.organization.grantsFunded.length < props.countGrantsFrom
const moreGrantsReceivedToLoad = data.organization.grantsReceived.length < props.countGrantsTo

if (moreGrantsFundedToLoad || moreGrantsReceivedToLoad) { 
  ...
```

Reply:
yes, good call - updated :+1: